jre/checks/oval/installed_app_is_java.xml

```diff
@@ -69,4 +70,10 @@
   <linux:rpminfo_state id="state_java_version" version="1">
     <linux:version datatype="version" operation="equals">1.8.0</linux:version>
   </linux:rpminfo_state>
+  <linux:rpminfo_state id="state_java_version_minimum" version="1">
+    <linux:version datatype="version" operation="greater than or equal">1.8.0</linux:version>
+  </linux:rpminfo_state>
+  <linux:rpminfo_state id="state_java_version_maximum" version="1">
+    <linux:version datatype="version" operation="less than">1.9.0</linux:version>
```

Comment:
I don't quite understand why we have this? Why do we care which JDK version they're running? Is it because our interior rules only apply on JDK8? Do we just need rules for JDK11+?

We care that, whatever major JDK version they have is the latest, but I don't quite understand why we should care which major version they're running --- and assume `test_oracle_java` only passes if it is JDK1.8.0 --- it should be independent of JDK versions :) 

My 2c. -- either test is misnamed (another PR) or these version checks should be removed altogether.

(I also don't use JRE checks so perhaps I'm completely off base here)

Reply:
@cipherboy yes, it's because our current ruleset applies to JDK version 8. This entire product was based on the Oracle JRE 8 STIG, and there are plans to do another round once a new STIG for ( I assume 11+) comes out. With that being said, I agree that we should probably start cleaning up our verbiage around this product. Perhaps make it `jre8` instead of `jre`, and start updating the test names so they make more sense of what is being checked.

Reply:
@carlosmmatos So I guess `test_oracle_java_rhel` should be updated as well?

Reply:
I don't even know if we offer/support that anymore..I guess I'm not that concerned with it as I doubt we are pumping out new packages for it. However, I'm also not opposed to just implementing the min/max check as the standard in the event that future openjdk packages start doing the same thing.

Reply:
Yeah, I didn't touch `test_oracle_java_rhel` as I don't have a testbed for that and while I believe it'd have the similar interesting things going on, I was operating under a "change as little as necessary to get my system detected".